ngraph/core/include/ngraph/pattern/op/label.hpp

```diff
@@ -118,10 +119,11 @@ namespace ngraph
                 {
                 }
                 Label(const Output<Node>& value)
-                    : Label(value.get_element_type(),
-                            value.get_partial_shape(),
-                            [](const Output<Node>&) { return true; },
-                            OutputVector{})
+                    : Label(
+                          value.get_element_type(),
+                          value.get_partial_shape(),
+                          [](const Output<Node>&) { return true; },
+                          OutputVector{})
```

Comment:
I don't know if I like this change or not, is there a way to use the old style in these cases?

Reply:
I didn't find style configuration flag to align this with the old style.